src/source/Dtls/Dtls.h

```diff
@@ -61,13 +61,13 @@ typedef struct {
 } DtlsSessionCertificateInfo, *PDtlsSessionCertificateInfo;
 
 typedef struct {
+    volatile BOOL isStarted;
```

Comment:
Does this need to be atomic?

Reply:
no it doesnt. it's always accessed within lock.